sub-packages/bionemo-geneformer/tests/bionemo/geneformer/scripts/test_train_geneformer.py

```diff
@@ -306,3 +360,64 @@ def test_limit_val_batches_is_int(required_args_reference, limit_val_batches):
     arglist = parse_kwargs_to_arglist(required_args_reference)
     parser = get_parser()
     parser.parse_args(arglist)
+
```

Comment:
code format checker probably will not like these empty lines, please install pre commit